Vagrantfile

```diff
@@ -40,53 +25,21 @@ sudo apt-get install -y zip unzip curl wget
 
 SCRIPT
 
-$mayadev = <<SCRIPT
+$mayascript = <<SCRIPT
 #!/bin/bash
 
 cd /opt/gopath/src/github.com/openebs/maya
 
 # Install dependencies required for Development
-bash scripts/install_go.sh
+bash buildscripts/install_go.sh
 
 # CD into the maya working directory when we login to the VM
 # A bit of conditional logic s.t. we do not repeat CD-ing
 grep "cd /opt/gopath/src/github.com/openebs/maya" /home/vagrant/.profile || \
   echo "cd /opt/gopath/src/github.com/openebs/maya" >> /home/vagrant/.profile
 
 echo "In-order to compile maya, look at various options provided in GNUmakefile"
-echo -e "\n\tTIP: Start with command:- make bootstrap"
-SCRIPT
-
-$mayamaster = <<SCRIPT
-#!/bin/bash
-
-cd /opt/gopath/src/github.com/openebs/maya
-make bootstrap 
-make dev
-maya setup-omm
-echo "export NOMAD_ADDR=http://172.28.128.3:4646" >> /home/vagrant/.profile
-echo "export MAPI_ADDR=http://172.28.128.3:5656" >> /home/vagrant/.profile
-
-SCRIPT
-
-$storagehost = <<SCRIPT
-#!/bin/bash
-
-echo "Testing output redirection"
-
-cd /opt/gopath/src/github.com/openebs/maya
-make bootstrap
-make dev
-maya setup-osh -omm-ips=172.28.128.3
-echo "export NOMAD_ADDR=http://172.28.128.3:4646" >> /home/vagrant/.profile
-
-SCRIPT
-
-
-$clientinstaller = <<SCRIPT
-
-sudo apt-get install -y open-iscsi fio
-
+echo -e "\n\tTIP: Start with command:- make"
```

Comment:
`make` will only build maya-cli right now(GNUMakefile), have to add `make apisever` in Vagrantfile to builld m-apiserver.

Reply:
by default `make` is running ` make test`

Reply:
@prateekpandey14 What should be the default target for make ?

Reply:
Sure, we can do this.